### PR TITLE
Josephwambura patch binding bugs

### PR DESCRIPTION
**Pull request by josephwambura**
A few fixes, first, it is some useless texts in the strings file (firestore project) and a bug found in the FilterDialog

### CODE REVIEW

**Comment by josephwambura** [comment]
@googlebot I signed it!

File: firestore/app/src/main/java/com/google/firebase/example/fireeats/kotlin/FilterDialogFragment.kt [hunk 1]
```diff
@@ -144,7 +144,7 @@ class FilterDialogFragment : DialogFragment() {
     }
 
     fun resetFilters() {
-        binding.let {
+        binding?.let {
```
Comment (rosariopfernandes) [inline firestore/app/src/main/java/com/google/firebase/example/fireeats/kotlin/FilterDialogFragment.kt:147]: In this case, `binding` is never null. In order to fix the error, we might want to use `_binding` instead.
```suggestion
        _binding?.let {
```

File: firestore/app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -61,7 +61,6 @@
         <item>Ramen</item>
         <item>Sushi</item>
     </string-array>
-    ite
```
Comment (rosariopfernandes) [inline firestore/app/src/main/res/values/strings.xml:64]: Good catch!

**Review by rosariopfernandes — CHANGES_REQUESTED** [review]

File: firestore/app/src/main/java/com/google/firebase/example/fireeats/kotlin/FilterDialogFragment.kt (hunk 1 above)
Comment (josephwambura) [reply firestore/app/src/main/java/com/google/firebase/example/fireeats/kotlin/FilterDialogFragment.kt:147]: Crashed if i used the implementation in a fragment, calling the Dialog from a fragment. But yeah _binding?.let is better

File: firestore/app/src/main/res/values/strings.xml (hunk 2 above)
Comment (josephwambura) [reply firestore/app/src/main/res/values/strings.xml:64]: Nice one

**Review by rosariopfernandes — APPROVED** [review]

**Comment by samtstern** [comment]
@josephwambura and @rosariopfernandes thank you both!

**Pull request merged**